apps/condo/domains/contact/components/ContactsEditor/index.tsx

```diff
@@ -240,9 +240,11 @@ export const ContactsEditor: React.FC<IContactEditorProps> = (props) => {
 
     const handleClickOnMinusButton = useCallback(() => {
         setDisplayEditableContactFields(false)
-        setSelectedContact(fetchedContacts[0])
+        if (!selectedContact) {
+            setSelectedContact(fetchedContacts[0])
```

Comment:
Is `fetchedContacts` always had length? Maybe do it a little safely?

Reply:
done